flake.nix

```diff
@@ -560,14 +554,9 @@
               installPhase = ''
                 mkdir $out
                 nixfmt --version
-
-                total_exit_code=0
-                for file in $(find ${all-directories-and-files} -type f -and -name "*.nix"); do
-                  echo "=== $file ==="
-                  nixfmt --check $file || total_exit_code=$?
-                  echo "==="
-                done
-                exit $total_exit_code
+                # note, really can just src with filer by .nix, no need all files 
+                nixfmt --check $(find ${all-directories-and-files} -name "*.nix" -type f | tr "\n" " ")
```

Comment:
the previous script printed what files were processed:
```
=== file1.nix ===
===
=== file2.nix ===
===
```
it can use as debug.
your script doesn't print what files are processed.

Reply:
if your script prints `""` in stdout, it could mean no errors or any file wasn't processed.

Reply:
so can echo find ${all-directories-and-files} -name "*.nix" -type f | tr "\n" " " ?

Reply:
yes, it will be useful

Reply:
will add echo in next PR